bazel/grpc_build_system.bzl

```diff
@@ -24,178 +24,203 @@
 #
 
 # The set of pollers to test against if a test exercises polling
-POLLERS = ['epollex', 'epollsig', 'epoll1', 'poll', 'poll-cv']
+POLLERS = ["epollex", "epollsig", "epoll1", "poll", "poll-cv"]
```

Comment:
Any reason for all the single-quote to double-quote changes in this PR? Skylark accepts either, correct? If Skylark prefers double-quote, let's keep the stylistic side of the PR separate from the functional change so that a single PR isn't cluttered with two types of changes.

Reply:
the clion blaze plugin seems to auto-lint, which causes all the formatting changes

Reply:
If you would like to exclude those changes I will manually reverse